.dockerignore

```diff
@@ -0,0 +1,12 @@
+*.swp
```

Comment:
Probably `*.sw[op]` for maximum coverage (assuming this sports regex like things like gitignore)